### ISSUE TITLE
Crash Diagnostics: Incorrect error when using vkGetSemaphoreCounterValue() to check command buffer completion

### ISSUE THREAD

**Issue by mbechard** [KhronosGroup/Vulkan-ValidationLayers#10060]
**Environment:**
 - OS:  Windows 11
 - GPU and driver version: RTX 6000 Ada, 572.93
 - SDK or header version if building from repo: 1.4.309.0
 - Options enabled (synchronization, best practices, etc.): Crash Diagnostics configuration

**Describe the Issue**

Similar to #1999, I'm checking to see if a command buffer is ready to be reset with call to `vkGetSemaphoreCounterValue()` for a timeline semaphore it is signalling. The Crash Diagnostic configuration produces the error when I then try to reset it:
```
ERROR : GENERAL - Message Id Number: 1 | Message Id Name: CDL
        Invalid Command Buffer Usage: Reset of VkCommandBuffer in use by GPU: uniqueID: 11, GX_Device::buildDefaultAttribBuffers: Primary Gfx (0x000001CFA60D6B40)
```

This error is not produced when using Validation configuration. If I instead use a wait() with a 0 timeout, the error is avoided.

You can reproduce the error by turning on the Crash Diagnostics and having it write to a log file, and then run TouchDesigner from here:
https://download.derivative.ca/TouchDesigner.2023.12230.exe

**Comment by spencer-lunarg** [comment]
@mbechard thanks for reporting this, but this should be reported to https://github.com/LunarG/CrashDiagnosticLayer

(I can't transfer it because its a LunarG repo and this is a KhronosGroup repo)

**Comment by mbechard** [comment]
Oops sorry! I'll move this and the other I made over there

**Comment by spencer-lunarg** [comment]
@mbechard thanks, I will still take a look (post upcoming SDK release next week) but likely the stuff CDL is injecting in invalid and triggering VVL

If you still want to use the Crash Diagnostic Layer, if you go `app` -> `VVL` -> `CDL` -> `driver` it should also still work

Using CDL layer and GPU-AV together might cause issues as we have similar issues with Renderdoc where it is hard for multiple tools/layer to try and inject/inspect things in the GPU command stream
